src/hotspot/share/prims/jvmti.xml

```diff
@@ -2987,8 +2987,8 @@ err = (*jvmti)-&gt;Deallocate(jvmti, stack_info);
       </parameters>
       <errors>
         <error id="JVMTI_ERROR_OPAQUE_FRAME">
-          Called or calling method is a native method.
-          The implementation is unable to pop this frame.
+          The implementation is unable to pop this frame
+          (e.g. called or calling method is a native method).
```

Comment:
Directly below we have special OPAQUE_FRAME language for virtual threads. Do we want to get rid of it since it is now covered by the above language? Note there is also language directly above that discusses suspended virtual threads.

Reply:
Thank you for the suggestion. Fixed now.